api/jobset/v1alpha2/jobset_types.go

```diff
@@ -22,6 +22,10 @@ import (
 const (
 	JobSetNameKey         string = "jobset.sigs.k8s.io/jobset-name"
 	ReplicatedJobReplicas string = "jobset.sigs.k8s.io/replicatedjob-replicas"
+	// GlobalJobReplicasKey is a label/annotation set to the total number of replicatedJob replicas.
+	// For each JobSet, this value will be equal to the sum of `replicas`, where `replicas`
+	// is equal to jobset.spec.replicatedJobs[*].replicas.
+	GlobalJobReplicasKey string = "jobset.sigs.k8s.io/global-job-replicas"
```

Comment:
`jobset.sigs.k8s.io/global-job-count`

Reply:
Actually, we used `jobset.sigs.k8s.io/replicatedjob-replicas` above, so the equivalent here would be `jobset.sigs.k8s.io/global-replicas` perhaps.

Reply:
Hi Abdullah.

Just to be extra sure. Are you suggesting to rename `jobset.sigs.k8s.io/global-job-replicas` to `jobset.sigs.k8s.io/global-replicas`? If so, I agree, it's a better name and I can change it : )

Reply:
Yes, to align with the other annotation we have for number of replicas within a replicatedJob

Reply:
Done!